tools/merger/lockfile

```diff
@@ -4,8 +4,5 @@ set -e
 cd "$(dirname "$0")"
 cd ../..
 
-BRANCH=`git rev-parse --abbrev-ref HEAD`
-if [[ $BRANCH == 'main' ]]; then
```

Comment:
How come this was removed?

Reply:
I _think_ it's the right piece to remove to ensure that CI always builds PR branches. But I may be wrong.

Reply:
I think it was the right piece.